kingfisher_scrapy/base_spider.py

```diff
@@ -592,7 +596,10 @@ def pages_url_builder(self, value, data, response):
     def limit_offset_range_generator(self, data, response):
         limit = self._resolve_limit(data)
         count = resolve_pointer(data, self.count_pointer)
-        return range(self.limit, count, limit)
+        if not self.yield_list_results:
+            return range(0, count, limit)
+        else:
+            return range(self.limit, count, limit)
```

Comment:
What is this fixing?

Reply:
the offset=0 is never downloaded otherwise, related to https://github.com/open-contracting/kingfisher-collect/issues/523 bullet 7

Reply:
Aha. To reduce duplication and explain the logic:

```suggestion
        # If `yield_list_results` is `True` (default), the response is parsed is `parse_list`.
        if self.yield_list_results:
            start = self.limit
        else:
            start = 0
        return range(start, count, limit)
```

Reply:
...and that is why I shouldn't work late and tired!